spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -42,7 +42,7 @@ def get_value(self, name):
         reason_big = _("The variable is too big to be retrieved")
         reason_not_picklable = _(
             "It was not possible to create a copy of the variable in the "
-            "kernel to pass it to Spyder."
+            "kernel to pass it to Spyder. the variable can't be pickled"
```

Comment:
```suggestion
            "kernel to pass it to Spyder.<br><br>"
            "If the object you're trying to view contains a generator, you "
            "need to replace it by a list because generators can't be "
            "serialized."
```

Improve this message to talk specifically about generators.